### PR TITLE
Add systemd unit files for hirte and hirte-agent

### PR DESCRIPTION
**Pull request by mwperina**
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: systemd-units/hirte.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Description=Hirte service orchestrator manager daemon
+Documentation=man:hirte(1) man:hirte.conf(5)
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/hirte
+
+[Install]
+WantedBy=multi-user.target
```
Comment (engelmi) [inline systemd-units/hirte.service]: Could you add a new line?

File: systemd-units/meson.build [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+install_data(
+  'hirte.service',
+  install_dir : systemd_unit_dir
+)
+
+install_data(
+  'hirte-agent.service',
+  install_dir : systemd_unit_dir
+)
```
Comment (engelmi) [inline systemd-units/meson.build]: Could you add a new line?

**Review by engelmi — APPROVED** [review]
LGTM.

**Review by mkemel — APPROVED** [review]

File: systemd-units/hirte.service (hunk 1 above)
Comment (mwperina) [reply systemd-units/hirte.service]: Done

File: systemd-units/meson.build (hunk 2 above)
Comment (mwperina) [reply systemd-units/meson.build]: Done

**Pull request merged**